lib/errors/index.js

```diff
@@ -224,21 +224,108 @@ exports.UnknownConstraintError = UnknownConstraintError;
  * Validation Error Item
  * Instances of this class are included in the `ValidationError.errors` property.
  *
- * @param {string} message An error message
- * @param {string} type The type of the validation error
- * @param {string} path The field that triggered the validation error
+ * @param {String} message      - An error message
```

Comment:
Not sure how esdocs reacts to this but it will be better to follow rest of code comment style and removed prefix `-`